### DISCUSSION TITLE
Up-to-date demo server?

### DISCUSSION THREAD

**Discussion by jaredcwhite** [ruby2js/ruby2js#73] [Ideas]
It looks like the demo site linked to from the docs is acting wonky: http://intertwingly.net/projects/ruby2js.cgi/all

Is there a way to fix this and to have this update automatically to the latest state of master for new commits? Maybe we migrate to something like Heroku? I'd be happy to help out with this if need be.

**Comment by rubys** [comment]
Yup, that certainly is wonky.  It doesn't seem to have the necessary gems installed.  I hadn't noticed as I've been using https://whimsy.apache.org/ruby2js/all which piggybacks on whimsy's infrastructure (i.e., it updates whenever something is pushed to the whimsy source tree).

If are willing to set something up with heroku, that's fine with me.

**Comment by jaredcwhite** [comment]
That sounds good…it's sort of related to the idea of setting up a new docs site too, so I'll probably try to tackle it as a unified project.

**Comment by rubys** [comment]
I've installed the gem on that machine, updated the load path, and pulled the latest code, but the site isn't set up to autoupdate.  A better solution would be appreciated.

**Comment by jaredcwhite** [comment]
Locking this convo, since it's all up-to-date now :+1:
